### PR TITLE
Updated OPtANNe PR

### PR DESCRIPTION
**Pull request by sourcesync**
@harsha-simhadri This PR should help us merge the OptANNe into main.  I've captured the essence of PRs 63 and 64 (originating from forked repos) here and if you accept/merge we can close 63 and 64.

### CODE REVIEW

File: algos.yaml
Comment (harsha-simhadri) [inline algos.yaml:574]: Could you please fix this.

File: .python-version
Comment (harsha-simhadri) [inline .python-version:1]: Is this needed?

File: algos.yaml
Comment (harsha-simhadri) [inline algos.yaml:756]: Clean up markers from merge conflicts and verify the contents?

File: benchmark/datasets.py
Comment (harsha-simhadri) [inline benchmark/datasets.py:165]: wouldn't this break main?

**Review by harsha-simhadri — CHANGES_REQUESTED** [review]

File: .python-version
Comment (sourcesync) [reply .python-version]: I don't think so...

File: algos.yaml
Comment (sourcesync) [reply algos.yaml:754]: yep

File: benchmark/datasets.py
Comment (sourcesync) [reply benchmark/datasets.py]: yes good catch

**Comment by sourcesync** [comment]
@harsha-simhadri I see you asked @sdongaonkar about PR63.  I think it would be best to wait for his response on that PR before merging this one, since he may be in a position to test on the hardware.

**Pull request closed without merging**
